### ISSUE TITLE
Having issues with trying to start mysql db with docker on windows 10

### ISSUE THREAD

**Issue by Riotsmurf** [docker-library/mysql#492]
I normally use linux, but since i got my laptop i will have to work with windows 10 as well.
I have googled around and cannot find a straight answer on why i cannot get this database to stay up.
I am familiar with using mysql, but not its errors how it needs to be setup.
I know this docker works fine on linux because i use it all the time in the office. I was wondering if anyone has some insight or ideas on how i can get this databse to stay up. 
Basically the situation is when i `docker-compose up` my app comes up, and it tries to start the database but errors out. 
Here is the log for that docker image.
[Image of  container log](https://i.imgur.com/hXnXTA5.png)

Here is my docker file.
[image of docker file](https://i.imgur.com/J97eV5S.png)

**Comment by Riotsmurf** [comment]
SOLUTION: for anyone who comes across this with the same issue. Try this. I ran this command `--innodb-use-native-aio=0`

In my docker-compose file it looks like this: 
```
conan_database:
    image: mysql:5.5
    command: --innodb-use-native-aio=0
    container_name: dconan_database_1
    ports:
    - "9000:3306"
    volumes:
    - ./conan_mysql:/var/lib/mysql
    environment:
    - MYSQL_RANDOM_ROOT_PASSWORD="yes"
```

**Comment by Riotsmurf** [comment]
Fuckin after 1 1/2 days of googling. Found it T_T ugh that was harder than it had to be.

**Comment by wglambert** [comment]
https://stackoverflow.com/questions/48239668/fails-to-initialize-mysql-database-on-windows-10

Yeah this seems to be an odd issue with windows 10 specifically, another windows 10 user had commented previously about this https://github.com/docker-library/mysql/issues/69#issuecomment-407334811

**Comment by Riotsmurf** [comment]
@wglambert Thanks for the link, thats exactly where i found my answer haha.
